app/cdap/components/ConnectionList/Components/HeaderSearch/index.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box } from '@material-ui/core';
+import Close from '@material-ui/icons/Close';
+import SearchRounded from '@material-ui/icons/SearchRounded';
+import HeaderSearchInputField from 'components/ConnectionList/Components/HeaderSearchInputField';
+import { useStyles } from 'components/ConnectionList/styles';
+import { IHeaderSearchProps } from 'components/ConnectionList/types';
+import React, { MouseEvent } from 'react';
+
+export default function({
+  eachFilteredData,
+  index,
```

Comment:
can we rename index to more specific if possible

Reply:
Done